### ISSUE TITLE
Swap Texture Pack makecertain blocks grey

### ISSUE THREAD

**Issue by Zachadactyl** [TheDuckCow/MCprep#282]
When I use "Swap Texture Pack" on an OBJ file from Mineways set o the recommended settings, some blocks such as grass, trees, and foliage.  I have tried Misa's Realistic pack, AVPBR, and VanillaAccurate texture packs all with no PBR, Specular, and SEUS.  None of these options work, but yet campfires, planks, torches, etc all work.  Why is this?

**Comment by TheDuckCow** [comment]
Are you using the in Mineways which exports textures into their own files in a textures directory? This is required for texture swap to work, otherwise the "all in one" picture will not work.

See [here for more info](https://youtu.be/o5CcG6idBws).

**Comment by Zachadactyl** [comment]
![image](https://user-images.githubusercontent.com/89289754/153097585-1d402ab0-9de2-4270-bc17-b14f736ebcab.png)

These are my settings for my export and it doesn't work

**Comment by Zachadactyl** [comment]
OH YES thx so much I could't find that Youtube video!

**Comment by Zachadactyl** [comment]
@TheDuckCow It is still not working after doing the YT vid at .25x speed...

**Comment by Zachadactyl** [comment]
I can get AVPBR to work now by hitting many buttons, I will keep trying.  Misa's and Faithful 2048x still are grey or don't import...

**Comment by Zachadactyl** [comment]
I am in both

**Comment by Zachadactyl** [comment]
Its always grass_block_top for 3+ packs now

**Comment by Zachadactyl** [comment]
Trees, grass, grass top, grass path.  It shows in render, and rendered mode, and material preview

**Comment by StandingPadAnimations** [comment]
Can you share a screenshot of what exactly happens in blender?

**Comment by Zachadactyl** [comment]
> Can you share a screenshot of what exactly happens in blender?

I will try

**Comment by Zachadactyl** [comment]
@TheDuckCow  and @StandingPadAnimations 

![Test_Normal](https://user-images.githubusercontent.com/89289754/153702391-8f011e6d-d2db-469d-b7c6-18d32beb3609.png)
![Test_Misas](https://user-images.githubusercontent.com/89289754/153702373-97ed04b3-122f-4c86-a205-1f1aac72f656.png)

The first image is my house (no judging pls it's awful) with normal MCPrep Default, and the second one is with Misa's 1.18b Texture Pack, notice the grey grass/grass_block_top, and odd tree colour.

**Comment by StandingPadAnimations** [comment]
Just curious, does this texture pack rely on any Optifine features? If so, it's not supported currently

**Comment by Zachadactyl** [comment]
Yes, it turns out it uses Optifine features, yet the main blocks that use it seem fine.  Will these be added in?  I sure hope so.  Many packs are Optifine only.

**Comment by StandingPadAnimations** [comment]
I think you can get around it as mentioned here: https://github.com/TheDuckCow/MCprep/issues/251

**Comment by Zachadactyl** [comment]
> I think you can get around it as mentioned here: #251

Seems advanced and confusing, but I will give it a try thanks!  I have many blocks to do...

**Comment by TheDuckCow** [comment]
Going to close this issue in favor of this other one listed, reminder you can always manually edit materials using the add color node that MCprep adds, and further more save your customized material into the "materials.blend" file in the folder of your custom resource pack. That lets you later "sync materials" in the future, [more info here](https://github.com/TheDuckCow/MCprep#sync-materials).
